src/index.html

```diff
@@ -4,10 +4,391 @@
+                specialists. Working directly with developers - so have
+                more useful information about the profitable deals.
+              </p>
+              <br>
+
+              <p>
+                <strong> We will never be late!</strong>
+              </p>
+              <p>
+                <strong>Guaranteed secure transaction - 100%</strong>
+              </p>
+            </div>
+          </div>
+        </div>
+      </section>
+
+      <section class="gallery">
+        <div class="container gallery__container">
+          <div class="gallery__title title">
+            <a name="gallery">
+              Gallery
+            </a>
+          </div>
+
+          <div class="gallery__pictures">
+            <div class="gallery__photos gallery__photos--left">
+              <img
+                src="./images/photos/1.jpg"
+                alt="marina palms"
+                class="gallery__img gallery__img--big"
+              >
+              <img
+                src="./images/photos/2.jpg"
+                alt="marina palms"
+                class="gallery__img gallery__img--small"
+              >
+            </div>
+            <p class="gallery__description">
+              <span  class="gallery__link">
+                <a href="#">Marina Palms</a>
+              </span>
+              / North Miami Beach, FL 33162
+            </p>
+
+            <div class="gallery__photos">
+              <img
+                src="./images/photos/3.jpg"
+                alt="Edition Residens"
+                class="gallery__img"
+              >
+              <img
+                src="./images/photos/4.jpg"
+                alt="Edition Residens"
+                class="gallery__img"
+              >
+            </div>
+            <p class="gallery__description">
+              <span class="gallery__link">
+                <a href="#">Edition Residens</a>
+              </span>
+              / Miami Beach, FL 33139
+            </p>
+
+            <div class="gallery__photos gallery__photos--right">
+              <img
+                src="./images/photos/5.jpg"
+                alt="Faena House"
+                class="gallery__img gallery__img--small"
+              >
+              <img
+                src="./images/photos/6.jpg"
+                alt="Faena House"
+                class="gallery__img gallery__img--big"
+              >
+            </div>
+            <p class="gallery__description">
+              <span class="gallery__link">
+                <a href="#">Faena House</a>
+              </span>
+              / Miami Beach, FL 33140
+            </p>
+          </div>
+
+          <button class="gallery__button">
+            View all
+          </button>
+
+        </div>
+      </section>
+
+      <section class="consultation">
+        <div class="container consultation__container">
+          <div class="consultation__title title">
```

Comment:
```suggestion
        <div class="consultation__container container">
          <div class="consultation__title title">
```
```suggestion
        <div class="container consultation__container">
          <div class="title consultation__title">
```
делай одинаково, первый вариант лучше